package.json

```diff
@@ -60,6 +60,8 @@
     "@curveball/problem": "^0.3.0",
     "@curveball/router": "^0.4.1",
     "@curveball/validator": "^0.9.0",
+    "dotenv": "^16.0.2",
+    "dotenv-defaults": "^5.0.2",
```

Comment:
I regret adding dotenv-defaults to our other packages. I wonder if we should just remove these?

Reply:
I'm not opposed to it!